settings.gradle

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+rootProject.name = 'static-code-analysis-tool'
+include 'ScanCommand'
+
```

Comment:
If I'm not mistaken, this is an additional new line. i.e. two lines at the end of the file. If that's the case, let's remove one of those two

Reply:
Fixed in [f569dee](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/f569deea8e438e558f4505d64002edaa768de6f0)